scripts/west_commands/runners/spi_burn.py

```diff
@@ -91,10 +107,11 @@ def _flash(self):
             # Run SPI burn flash tool
             self.check_call(cmd_flash)
 
-        finally:
+        except RuntimeError as e:
+            print(e)
```

Comment:
Why was this added in the latest push? Please remove it; we don't leave debug printlines around like this.

Reply:
Removed